terraform/ec2/mac/main.tf

```diff
@@ -45,7 +45,7 @@ resource "aws_instance" "cwagent" {
   }
 
   tags = {
-    Name = "cwagent-integ-test-ec2-${var.test_name}-${module.common.testing_id}"
```

Comment:
Since the way we use it is to inject [the name and the os](https://github.com/aws/amazon-cloudwatch-agent/blob/main/.github/workflows/integrationTest.yml#L482) or only the os which is not useful for the name. Deleting it.

Reply:
Will there only be 1 mac test? I think the answer is yes.

Reply:
Yes. However, `${module.common.testing_id}` make it unique enough.